### PR TITLE
Update feedback links

### PR DESCRIPTION
**Pull request by edwardhorsford**
This update:
* removes the feedback link from the registration form
* removes the feedback link from the qts recommend form
* updates the phase banner to only ask about feedback
* update our footer to have two sections
* misc updates to links in footer

New phase banner:
Screenshot 2022-04-14 at 10 05 11

Updated footer:
Screenshot 2022-04-14 at 15 18 15

### CODE REVIEW

File: app/views/_includes/footer-settings.njk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        </a>
+      </li>
+
+      <li class="govuk-footer__inline-list-item">
+        <form method="post" action="/admin/clear-data" class="app-hidden-form">
+          <button class="app-link-button">
+            Clear data
+          </button>
+        </form>
+
```
Comment (ralph-hawkins) [inline app/views/_includes/footer-settings.njk]:
```suggestion
```

File: app/views/_includes/footer-settings.njk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<div class="govuk-footer">
+  <div class="govuk-width-container">
```
Comment (ralph-hawkins) [inline app/views/_includes/footer-settings.njk]:
```suggestion
<div class="govuk-footer">
  <div class="govuk-width-container">
```

File: app/views/layout.html [hunk 3]
```diff
@@ -179,51 +179,54 @@
...
-  </form>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      <h2 class="govuk-heading-s">
+        Get support
+      </h2>
+
+      <ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16">
+        <li><a class="govuk-link govuk-footer__link" href="mailto:becomingateacher@digital.education.gov.uk">becomingateacher<wbr>@digital.education.gov.uk</a></li>
+        <li>We aim to respond within 5 working days, or one working day for more urgent&nbsp;queries</li>
```
Comment (ralph-hawkins) [inline app/views/layout.html:190]:
```suggestion
        <li>We aim to respond within 5 working days, or one working day for more <span class="app-nowrap">urgent queries</span></li>
```

File: app/views/layout.html [hunk 4]
```diff
@@ -179,51 +179,54 @@
...
+      </ul>
+    </div>
+
+    <div class="govuk-grid-column-one-half">
+      <h2 class="govuk-heading-s">
+        Give feedback
+      </h2>
+
+      <p class="govuk-body govuk-!-font-size-16"><a href="/guidance" class="govuk-link govuk-footer__link">Give feedback to help us improve Register trainee teachers</a></p>
```
Comment (ralph-hawkins) [inline app/views/layout.html]:
```suggestion
      <p class="govuk-body govuk-!-font-size-16"><a href="/guidance" class="govuk-link govuk-footer__link">Give feedback to help us improve <span class="app-nowrap">Register trainee teachers</span></a></p>
```

File: app/views/layout.html (hunk 3 above)
Comment (ralph-hawkins) [inline app/views/layout.html:190]: Is it weird that we say ‘`5` working days’ and ‘`one` working day’?

**Review by ralph-hawkins — APPROVED** [review]

File: app/views/layout.html (hunk 3 above)
Comment (edwardhorsford) [reply app/views/layout.html:190]: @MaeveRoseveare ?

Comment (MaeveRoseveare) [reply app/views/layout.html:190]: In the GOV.UK style guide, for numbers we would write out 'one' and use numerals for anything higher than one. Expect if it's part of a step or a list. https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style (under 'N' > Numbers)

**Pull request merged**